src/leases/components/leaseSections/summary/LeaseHistoryEdit.js

```diff
@@ -156,12 +178,122 @@ class RelatedLeasesEdit extends Component<Props, State> {
     } = this.props;
     const {
       newLease,
-      relatedLeasesAll,
+      leaseHistoryItemsAll,
       relatedLeasesFrom,
       relatedLeasesTo,
       stateOptions,
     } = this.state;
 
+    const handleDelete = this.handleDelete
+    const handleDeleteRelatedPlotApplication = this.handleDeleteRelatedPlotApplication
+
+    const renderLeaseWithPlotSearchesAndApplications = (lease, active) => {
+      const historyItems = []
+
+      if (lease.plot_searches.length) {
+        lease.plot_searches.forEach((plotSearch) => {
+          historyItems.push({
+            key: `plot-search-${plotSearch.name}-${Math.random().toString()}`,
+            id: plotSearch.id,
+            itemTitle: plotSearch.name,
+            startDate: plotSearch.begin_at,
+            endDate: plotSearch.end_at,
+            plotSearchType: plotSearch.type,
+            plotSearchSubtype: plotSearch.subtype,
+            itemType: "Haku",
```

Comment:
This `itemType` was also used in another file. I wondered whether it is shown to the user. But since it is here again, how about making it enum?